src/scripts.js

```diff
@@ -130,20 +146,20 @@ const createRecipeCardEventListener = () => {
   const recipeCards = document.querySelectorAll('article');
   recipeCards.forEach(recipeCard => {
     recipeCard.addEventListener('click', (event) => {
+      console.log(event.target);
       displayBigModal(event);
-      console.log(event.target.id);
     });
   });
 }
 
 const displayBigModal = (event) => {
-  removeClass([bigModal], 'hidden');
+  removeClass([bigModal, homeButton], 'hidden');
   addClass([mainDisplay], 'hidden');
   populateBigModal(event);
 }
 
 const populateBigModal = (event) => {
-  let selectedRecipe = cookbook.currentRecipes.find(recipe => recipe.id === parseInt(event.target.parentNode.id));
+  let selectedRecipe = cookbook.currentRecipes.find(recipe => recipe.id === parseInt(event.target.id));
```

Comment:
Good catch recognizing the event.target.id was not coming out as the same data type and needed to be parsed